package.yaml

```diff
@@ -26,16 +26,15 @@ dependencies:
   - base == 4.*
   - bytestring == 0.10.*
   - containers >= 0.5 && < 0.7
-  - data-fix == 0.0.7 || == 0.2.0
+  - data-fix ^>= 0.3
   - directory == 1.3.*
   - filepath == 1.4.*
-  - hnix == 0.6.*
+  - hnix ^>= 0.10
   - mtl == 2.2.*
-  - prettyprinter == 1.2.*
+  - prettyprinter ^>= 1.7
```

Comment:
are we incompatible with 1.2 now?

Reply:
No, but HNIX is and when running `nix-build` some haskellPackages infrastructure complained about inconsistent dependencies across dependency tree without constraints at some point.

Reply:
Hmmmm… seems there's maybe a difference between no constraints and some constraints? Relaxed them to be compatible with the old ones.